geopyspark/geotrellis/rdd.py

```diff
@@ -120,6 +120,32 @@ def to_numpy_rdd(self):
         ser = self.geopysc.create_tuple_serializer(result._2(), value_type=TILE)
         return self.geopysc.create_python_rdd(result._1(), ser)
 
+    def to_tiled_rdd(self, extent=None, layout=None, crs=None, tile_size=256,
```

Comment:
I think so. The reason being is that I try not to refer `TiledRasterRDD`s as `layer`s for the sake of simplicity (though there are a few inconsistencies in the docs). Plus, it matches the return type more closely in terms of name.